Presentation.Web/app/models/ui-customization/ui-customization.ts

```diff
@@ -9,12 +9,12 @@
     }
 
     export interface IUICustomizationTreeVisitor {
-        visitNode(node: UINode);
+        visitNode(node: UINode, state?: boolean);
```

Comment:
no no no ... why are you changing the signature?
That parameter is supposed to go in the ctor of the vistor instance